### PR TITLE
Add Versioning Use Case 

### PR DESCRIPTION
**Pull request by SteveLasker**
Provides baseline discussion for clarifying [requirements around feeds](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/11).  
- See [Refine Definition of Feed #11](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/11)

### CODE REVIEW

File: draft-ietf-scitt-software-use-cases.md [hunk 1]
```diff
@@ -336,6 +336,28 @@ There is no standardized way to:
 * notify software integrators of vulnerabilities identified during security scans of running software
 * provide valid annotations on build integrity to ensure conformance
 
+## Identify Statements and Updates to Specific Versions of Released Software
+
+Software producers often have multiple and concurrent supported versions of a product.
+The versions may represent major feature or compatibility differentiating releases (1.0, 2.0), or implementations for different Operating System Platforms and Architectures (Linux, Mac, Windows, AMD, ARM, x86, x64).
+
+For each release, the software producer must be capable of providing statements, unique to that version.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-software-use-cases.md:344]: `statement` drifts in into the scitt terminology a bit. How about

```suggestion
For each release, the software producer must be capable of providing assertions, unique to that version.
```

File: draft-ietf-scitt-software-use-cases.md [hunk 2]
```diff
@@ -336,6 +336,28 @@ There is no standardized way to:
...
+
+
+
+Consumers and Producers want:
+
+* a standard means to associate vulnerability information, statements of quality, statements of support and end of life (EOL) with a specific version of a product
+* a standard means to identify a patched version, specific to their Operating System and Platform
+* a standard means to differentiate major and minor version upgrades
+* a standard means to provide concurrent versioned updates
+
```
Comment (henkbirkholz) [inline draft-ietf-scitt-software-use-cases.md:360]: There is no `There is no standardized way to:` paragraph/subsection.

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
Minor nits and there is no `There is no standardized way to:` paragraph/subsection included in the proposed use case

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-software-use-cases.md (hunk 2 above)
Comment (SteveLasker) [reply draft-ietf-scitt-software-use-cases.md:360]: I replaced the `Consumers and Producers want:` with `There is no standardized way to:` heading.
We may want to update this further.

**Pull request merged**